packages/icons/icons.json

```
@@ -4541,5 +4541,19 @@
         "tags": "arrow, box, object, counter-clockwise, ccw, left",
         "group": "interface",
         "codepoint": 62345
+    },
+    {
+        "displayName": "Sort: ascending filter",
+        "iconName": "sort-asc-filter",
```

Can we name these so that the primary object is listed first and the modifier is a suffix?
```suggestion
        "iconName": "filter-sort-asc",
```
This would match up better with some of the other preexising icon naming conventions for filter like `filter-keep`, `filter-open`, `filter-remove`, etc.

https://github.com/palantir/blueprint/blob/7e5903dca42c763d3654e66691ffbd5702c7a558/packages/icons/icons.json#L2865-L2878